ports/servoshell/minibrowser.rs

```diff
@@ -18,14 +23,17 @@ use servo::webrender_surfman::WebrenderSurfman;
 use crate::browser::Browser;
 use crate::egui_glue::EguiGlow;
 use crate::events_loop::EventsLoop;
+use crate::geometry::winit_position_to_euclid_point;
 use crate::parser::location_bar_input_to_url;
 use crate::window_trait::WindowPortsMethods;
 
 pub struct Minibrowser {
     pub context: EguiGlow,
     pub event_queue: RefCell<Vec<MinibrowserEvent>>,
-    pub toolbar_height: Cell<Length<f32, DeviceIndependentPixel>>,
+    pub toolbar_height: Length<f32, DeviceIndependentPixel>,
+    widget_surface_fbo: gl::GLuint,
```

Comment:
Why not store a `NativeFramebuffer` here to avoid having to `map(...)` it every time?

Reply:
done in f6f576b20cbdafa4765c66ac400894be392caa91